### ISSUE TITLE
[FKA]“Typing mode enabled” Alert  is not provided when pressing Tab after activating Password TextField with Space

### ISSUE THREAD

**Issue by preethi0228-git** [flutter/flutter#192178]
### Steps to reproduce

1. Clone the a11y assessments app from flutter repo and build the app in iPad/iPhone
2. Enable Full Keyboard Access on the iPad.
3. Connect a physical keyboard.
4. Launch the application.
5. Navigate to the Password TextField using the Tab key.
6. Once the Password TextField receives focus, press Space to activate it.
7. Press the Tab key while the focus is on the Password TextField.

### Expected results

When the Password TextField is focused and the user presses Tab, the appropriate “typing mode enabled” announcement should be provided, indicating that the text field is in edit mode. Like other text fields 

### Actual results

After the Password TextField is activated using the Space key, pressing Tab does not provide the “typing mode enabled” announcement, even though the focus is on the Password TextField. 

### Code sample

<details open><summary>Code sample</summary>

```dart
[Paste your code here]
```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

 
https://github.com/user-attachments/assets/a84c7be3-12ce-4d5c-9327-6d4c0aa1cf91

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
[✓] Flutter (Channel master, 3.48.0-1.0.pre-524, on macOS 27.0 26A5421a darwin-arm64, locale en-US) [1,364ms]
    • Flutter version 3.48.0-1.0.pre-524 on channel master at /Users/preethikandukuri/development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f5f6313e8c (4 hours ago), 2026-08-31 05:22:09 +0000
    • Engine revision f5f6313e8c
    • Dart version 3.14.0 (build 3.14.0-180.0.dev)
    • DevTools version 2.61.0-dev.0
    • Feature flags: enable-web, enable-linux-desktop, enable-macos-desktop, enable-windows-desktop, enable-android,
      enable-ios, cli-animations, enable-native-assets, enable-record-use, enable-swift-package-manager,
      omit-legacy-version-file, enable-lldb-debugging, enable-uiscene-migration, enable-riscv64, enable-hcpp

[✓] Android toolchain - develop for Android devices (Android SDK version 37.0.0) [455ms]
    • Android SDK at /Users/preethikandukuri/Library/Android/sdk
    • Emulator version 37.1.11.0 (build_id 15917651) (CL:N/A)
    • Platform android-37.0, build-tools 37.0.0
    • ANDROID_HOME = /Users/preethikandukuri/Library/Android/sdk
    • Java binary at: /Applications/Android Studio RC.app/Contents/jbr/Contents/Home/bin/java
      This JDK is specified in your Flutter configuration.
      To change the current JDK, run: `flutter config --jdk-dir="path/to/jdk"`.
    • Java version OpenJDK Runtime Environment (build 25.0.3+-15898627-b508.16)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 27.0) [657ms]
    • Xcode at /Applications/Xcode-beta276.app/Contents/Developer
    • Build 27A5252f
    • CocoaPods version 1.16.2

[✓] Chrome - develop for the web [4ms]
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Connected device (5 available) [6.0s]
    • sdk gphone16k arm64 (mobile) • emulator-5554                        • android-arm64  • Android 17 (API 37)
      (emulator)
    • iPad (wireless) (mobile)     • 00008122-001A08DC22B9801C            • ios            • iOS 27.0 24A5424a
    • iPhone 17 Pro (mobile)       • 6A22C9A6-0DD2-4ADA-84BB-ECB0DDE39E6C • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-27-0 (simulator)
    • macOS (desktop)              • macos                                • darwin-arm64   • macOS 27.0 26A5421a
      darwin-arm64
    • Chrome (web)                 • chrome                               • web-javascript • Google Chrome 152.0.7977.65

[✓] Network resources [893ms]
    • All expected network resources are available.

```

</details>

**Comment by okorohelijah** [comment]
from triage: @preethi0228-git can you try reproing with voice-over and FKA enabled? Also can you repro with just voice-over enabled and fka disabled

**Comment by preethi0228-git** [comment]
Hi @okorohelijah ,

Verified both scenarios as requested:

VoiceOver ON + Full Keyboard Access ON – Reproduced the issue. After activating the Password TextField with Space and pressing Tab, I do not hear the “typing mode enabled” announcement.
VoiceOver ON + Full Keyboard Access OFF – I also do not hear the “typing mode enabled” announcement after pressing Tab.

So, with VoiceOver enabled, I’m not receiving the “typing mode enabled” announcement in either configuration.

Could you please confirm whether the announcement is expected specifically with VoiceOver + FKA enabled, or if there is any additional VoiceOver setting/configuration required for this announcement?

https://github.com/user-attachments/assets/66a8e414-7560-4b65-8d0e-dc0e949d5b2f

 

https://github.com/user-attachments/assets/42666785-84de-4811-86c9-9918bdfb72bd



Thanks
